cp-quickstart/start-docker-cloud.sh

```diff
@@ -5,95 +5,169 @@
+
+printf "Obtaining the ksqlDB App Id\n"
+CMD="ccloud ksql app list -o json | jq -r '.[].id'"
+ksqlAppId=$(eval $CMD) \
+  && print_code_pass -c "$CMD" -m "$ksqlAppId" \
+  || exit_with_error -c $? -n "$NAME" -m "$CMD" -l $(($LINENO -3))
+
+printf "\nConfiguring ksqlDB ACLs\n"
+CMD="ccloud ksql app configure-acls $ksqlAppId pageviews users PAGEVIEWS_FEMALE pageviews_female_like_89"
+$CMD \
+  && print_code_pass -c "$CMD" \
+  || exit_with_error -c $? -n "$NAME" -m "$CMD" -l $(($LINENO -3))
+
+sleep 60
+printf "\nSubmitting KSQL queries via curl to the ksqlDB REST endpoint\n"
+printf "\tSee https://docs.ksqldb.io/en/latest/developer-guide/api/ for more information\n"
+while read ksqlCmd; do # from docker-cloud-statements.sql
+	response=$(curl -w "\n%{http_code}" -X POST $KSQL_ENDPOINT/ksql \
+	       -H "Content-Type: application/vnd.ksql.v1+json; charset=utf-8" \
+	       -u $KSQL_BASIC_AUTH_USER_INFO \
+	       --silent \
+	       -d @<(cat <<EOF
+	{
+	  "ksql": "$ksqlCmd",
+	  "streamsProperties": {
+			"ksql.streams.auto.offset.reset":"earliest",
+			"ksql.streams.cache.max.bytes.buffering":"0"
+		}
+	}
 EOF
-))
-  echo $response
-  if [[ ! "$response" =~ "SUCCESS" ]]; then
-    echo -e "\nWARN: KSQL command '$ksqlCmd' did not include \"SUCCESS\" in the response. Please troubleshoot."
-  fi
-done <statements.sql
-
-echo "-----------------------------------------------------------"
-
-echo
-echo "Local client configuration file written to $CONFIG_FILE"
-echo
-echo "Confluent Cloud KSQL is running and accruing charges. To destroy this demo run and verify ->"
-echo "    ./stop-docker-cloud.sh $CONFIG_FILE"
-echo
+	))
+	echo "$response" | {
+		echo "$reponse" >> $LOG_FILE
+	  read body
+	  read code
+	  if [[ "$code" -gt 299 ]];
+	    then print_code_error -c "$ksqlCmd" -m "$(echo "$body" | jq .message)"
+	    else print_code_pass  -c "$ksqlCmd" -m "$(echo "$body" | jq -r .[].commandStatus.message)"
+	  fi
+	}
+sleep 3;
+done < docker-cloud-statements.sql
+printf "\nksqlDB ready\n"
+
+printf "\nLocal client configuration file written to $CONFIG_FILE\n\n"
+
+printf "====== Verify\n"
+printf "If you have a local install of the Confluent Platform, you can run the following commands to view streaming data formatted in Avro and Protobuf. (If you do not have the tools, you can follow the Confluent Platform quick start to obtain them: https://docs.confluent.io/current/quickstart/ce-quickstart.html)\n"
+
+# Note: The values which are substituted here are sourced into the environment above
+#		where we call source ./delta_configs/env.delta
+printf "\nTo view the Protobuf formatted data in the users topic:\n\t";print_code "kafka-protobuf-console-consumer --consumer.config ./delta_configs/ak-tools-ccloud.delta --bootstrap-server $BOOTSTRAP_SERVERS --property schema.registry.url=$SCHEMA_REGISTRY_URL --property schema.registry.basic.auth.user.info=$SCHEMA_REGISTRY_BASIC_AUTH_USER_INFO --property basic.auth.credentials.source=USER_INFO --topic users"
```

Comment:
Users of this demo will not have CP install, so this command has to be provided by a Docker container.